uhubctl.c

```diff
@@ -656,6 +661,11 @@ static int usb_find_hubs()
                         info.actionable = 0;
                     }
                 }
+                if (opt_level != -1){
```

Comment:
Can you please put a space before `{`? This code should use consistent formatting as other code is using.